typescript/alert-center/README.md

```diff
@@ -0,0 +1,46 @@
+# DBOS Alert Response Center
+
+This app demonstrates producing and consuming Kafka messages from DBOS. A Kafka message creates an alert. Alerts are placed into a database queue. Employees can log in to the portal and respond to any outstanding alerts. After an employee gets an assignment, they have 30 seconds to fix the issue! After the time expires, the alert may be reassigned to a different employee. All incoming alert messages are handled exactly once.
+
+## Running up the app locally
+
+Make sure you have a local postgres setup and Docker installed, as shown in the [Quickstart](https://docs.dbos.dev/quickstart?language=typescript). Then:
+
+Using Docker, start a Kafka broker with the provided file:
+```shell
+cd alert-center
+export KAFKA_BROKER="localhost:9092"
+docker-compose -f kafka-compose.yml up
+```
+This will start a session with terminal output. You can leave it running.
+
+Then, build, migrate and run the app:
+```shell
+cd alert-center
+export KAFKA_BROKER="localhost:9092"
+export PGPASSWORD="..." #export your password if using Docker for Postgres
+npm install
+npm run build
+npx dbos migrate
+
```

Comment:
Might also add directions for running this on DBOS Cloud (given you have a Kafka somewhere)

Reply:
Should be addressed